src/main/java/com/google/graphgeckos/dashboard/api/DashboardController.java

```diff
@@ -0,0 +1,45 @@
+package com.google.graphgeckos.dashboard.api;
+
+import com.google.graphgeckos.dashboard.datatypes.BuildInfo;
+import com.google.graphgeckos.dashboard.storage.DatastoreRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please [do not use wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
Done!